intg/src/main/java/org/apache/atlas/model/discovery/ElasticsearchMetadata.java

```diff
@@ -0,0 +1,37 @@
+package org.apache.atlas.model.discovery;
+
+import org.apache.commons.collections.MapUtils;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class ElasticsearchMetadata {
+
+    private Map<String, List<String>> highlights;
+
+    public Map<String, List<String>> getHighlights() {
+        return highlights;
+    }
+
+    public void setHighlights(Map<String, List<String>> highlights) {
+        this.highlights = highlights;
+    }
+
+    public void addHighlights(Map<String, List<String>> highlights) {
```

Comment:
`setHighlights` and `addHighlights` seems redundant. Please use `addHighlights` only

Reply:
`setHighlights ` removed